src/resources/MachineLearning/ModelInformation/ModelInformationInterfaces.ts

```diff
@@ -29,7 +29,8 @@ export interface ModelInformationART {
             filters: any;
             words: number;
             stopwords: number;
-            [otherKeys: string]: number;
+            docPerFilters: Record<string, number>;
+            [otherKeys: string]: number | Record<string, number>;

```

Comment:
Maybe we could be more flexible here 🤷 
```suggestion
            [otherKeys: string]: any;
```

Reply:
lol good idea